k8s/core/configmap/configmap_lock_v2.go

```diff
@@ -387,20 +387,27 @@ func (c *configMap) refreshLock(id, key string) {
 		select {
 		case <-refresh.C:
 			lock.Lock()
-
+			isConflictErrCount := 0
 			for !lock.unlocked {
 				c.checkLockTimeout(c.defaultLockHoldTimeout, startTime, id)
 				currentRefresh = time.Now()
 				if _, err := c.tryLock(id, key, true); err != nil {
+					if k8s_errors.IsConflict(err) {
+						isConflictErrCount++
+						if isConflictErrCount%10 == 0 {
```

Comment:
nit..Instead of hardcoding 10 let's use a global variable..numRetryCount or numConflictErrorRetryCount o

Reply:
Very specific to a logging pattern so I'm keeping it hardcoded